src/session.ts

```diff
@@ -17,10 +17,12 @@ import {
     ResolvedTransaction,
     SigningRequest,
 } from 'eosio-signing-request'
+import { Account } from '@wharfkit/account'
+
 import zlib from 'pako'
 
-import {ChainDefinition, ChainDefinitionType, Fetch} from './types'
-import {getFetch} from './utils'
+import { ChainDefinition, ChainDefinitionType, Fetch } from './types'
```

Comment:
Not sure why our IDEs are formatting code differently.. I am using CS Code now 😅

Reply:
Regarding brackets: you can try adding this to the end of the eslint file. Current set to warn. 
```
prettierOptions: {
    bracketSpacing: false,
},
```

Reply:
Using `make format` seems to follow the rules. One thing to check with VSCode is that you have the Prettier extension installed:

<img width="518" alt="image" src="https://user-images.githubusercontent.com/677686/211910624-9153fc52-d5c2-4b8b-bc73-daea1ebd2693.png">

I'm not sure whether or not it obeys those rules by default. We do have a `.prettierrc` file in the root that your editor should follow:

https://github.com/wharfkit/session/blob/83b598de7f1512f758f8b5cb768de9d49846bc1b/.prettierrc#L1-L8